spec/features/reservations_spec.rb

```diff
@@ -549,7 +561,7 @@
   context 'valid items on confirmation page' do
     before(:each) do
       empty_cart
-      select('10', from: 'items_per_page') # show more than 1 item (default)
+      allow(@app_config).to receive(:override_on_create).and_return(false)
```

Comment:
If it isn't mocked it returns the mocked object, which evaluates to true; it won't load the request page if necessary because override will be enabled. I guess it should probably move to the shared examples that expect the request page?